src/NetMQ.Tests/RouterTests.cs

```diff
@@ -1,17 +1,14 @@
 ﻿using System;
 using NUnit.Framework;
+using System.Text;
 
-namespace NetMQ.Tests
-{
+namespace NetMQ.Tests {
```

Comment:
Sure! Actually that test was not related to the PR, but to another issue I asked about. I accidentally added it.